apps/meteor/server/sdk/types/IQueueWorkerService.ts

```diff
@@ -1,3 +1,10 @@
+export type HealthAggResult = {
+	total: number;
+	type: string;
+	status: 'Rejected' | 'In progress';
```

Comment:
Can we make this enum pls? Just to avoid typo's

Reply:
typos would produce a type error anyways, but i can make it 👀